astropy/table/table.py

```diff
@@ -1711,7 +1667,8 @@ def index_column(self, name):
         except ValueError:
             raise ValueError("Column {0} does not exist".format(name))
 
-    def add_column(self, col, index=None, name=None, rename_duplicate=False, copy=True):
+    def add_column(self, col, index=None, name=None, rename_duplicate=False, copy=True,
+                   default_name=None):
```

Comment:
Should update the docstring to note that `col` does not have to be a `Column` instance.